src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -550,6 +550,21 @@
           "default": {}
         },
 
+        "stdin": {
+          "description": "A file or portion of a file containing the standard input stream to the process that was invoked.",
+          "$ref": "#/definitions/physicalLocation"
+        },
+
+        "stdout": {
+          "description": "A file or portion of a file containing the standard output stream from the process that was invoked.",
+          "$ref": "#/definitions/physicalLocation"
+        },
+
+        "stderr": {
+          "description": "A file or portion of a file containing the standard error stream from the process that was invoked.",
+          "$ref": "#/definitions/physicalLocation"
+        },
+
```

Comment:
This feature was revised twice since it was first introduced by oasis-tcs/sarif-spec#92: "Add stdin/stdout/stderr on invocation":
- oasis-tcs/sarif-spec#110: "Specify how to treat a file that contains interleaved stdout/stderr" -- we introduced the `stdoutStderr` property
- oasis-tcs/sarif-spec#133: "Make stdin/stdout/stderr/stdoutStderr fileLocation instead of physicalLocation"

So let's go straight to the [final version](https://github.com/oasis-tcs/sarif-spec/blob/fc429120eb49dcd05e6df126c969d6f4447d5a3a/Schemata/sarif-schema.json#L532) (Lines 532-550). The differences from what you have are:
1. Remove "or portion of a file" from the descriptions.
2. Change the type from `physicalLocation` to `fileLocation`.
3. Add `stdoutStderr`.